### PR TITLE
Included factorial program in C language

### PR DESCRIPTION
**Pull request by YenruoY**
**Please describe your program and how to run it.**
Factorial program that takes a number as a command line argument and prints the factorial 

--------
**What Programming Language?**

Language Used : C
Compiler used : GCC

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -361,7 +361,12 @@ Place: Croatia </br>
...
 Email: antonio32a.1337@gmail.com </br>
 
-Name: [BenjaminUrquhart](https://github.com/BenjaminUrquhart)<br>
-Coding Experience: Discord Bot Developer (Java) + misc. wrappers. <br>
-Programming Languages: Python 2/3, Java 8<br>
-Email: benjamin@ericsart.com
+Name: [BenjaminUrquhart](https://github.com/BenjaminUrquhart)</br>
+Coding Experience: Discord Bot Developer (Java) + misc. wrappers. </br>
+Programming Languages: Python 2/3, Java 8</br>
+Email: benjamin@ericsart.com</br>
```
Comment (wzhouwzhou) [inline CONTRIBUTORS.md]: To avoid merge conflicts, would you mind reversing these changes?

**Review by wzhouwzhou — CHANGES_REQUESTED** [review]

**Review by YenruoY — COMMENTED** [review]
I think its done

**Review by wzhouwzhou — APPROVED** [review]
Looks good now, thanks!

**Comment by wzhouwzhou** [comment]
Why was this closed?

**Pull request closed without merging**